src/DropDownMenu/DropDownMenu.js

```diff
@@ -241,7 +261,7 @@ class DropDownMenu extends Component {
 
   handleTouchTapControl = (event) => {
     event.preventDefault();
-    if (!this.props.disabled) {
+    if (!this.props.disabled && !(this.props.multiple && this.state.open)) {
```

Comment:
When multiple is true, the dropdown shouldn't be closed on touch. We wan't to leave it open for further selections.